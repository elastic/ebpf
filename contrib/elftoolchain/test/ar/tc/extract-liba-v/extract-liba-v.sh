# $Id$
inittest extract-liba-v tc/extract-liba-v
extshar ${TESTDIR}
extshar ${RLTDIR}
runcmd "${AR} vx liba.a" work true
rundiff true
