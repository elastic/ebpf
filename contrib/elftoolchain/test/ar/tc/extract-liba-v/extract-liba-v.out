x - a1.o
x - a2.o
x - a3.o
x - a4.o
